src/library/scala/collection/mutable/IndexedSeq.scala

```diff
@@ -17,6 +17,13 @@ trait IndexedSeqOps[A, +CC[_], +C <: AnyRef]
   extends scala.collection.IndexedSeqOps[A, CC, C]
     with SeqOps[A, CC, C] {
 
+  def mapInPlace(f: A => A): this.type = {
```

Comment:
There is no unit test for this?

Reply:
We have this partest: https://github.com/scala/scala/blob/2.13.x/test/files/run/transform.scala#L7